core/deployment/src/main/java/io/quarkus/deployment/builditem/NativeAccessBuildItem.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class NativeAccessBuildItem extends SimpleBuildItem {
```

Comment:
Do we consider this build item as a flag, or should the extension producing it pass some data (like a path or something)